tools/subgraph-example/subgraph/subgraph.yaml

```diff
@@ -0,0 +1,69 @@
+specVersion: 0.0.5
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: ExampleERC721
+    network: local
```

Comment:
Q: Does this work is pointing to previewnet for example and if so do you change the network name?

Reply:
If you change the `network` property it should also change the network which you want to target - https://github.com/graphprotocol/graph-node/blob/master/docs/subgraph-manifest.md#15-data-source

If you run the graph-node using the `docker-compose` file, I think it only supporst a single network and you have to change this line `https://github.com/graphprotocol/graph-node/blob/master/docker/docker-compose.yml#L22`, where you need to replace `mainnet` with the name of the network, and the url with the network endpoint. 

If you buid it from source, you can specify multiple networks https://github.com/graphprotocol/graph-node/blob/master/docs/config.md#configuring-ethereum-providers, then just chaning the `network` property to the ones of the the specified should tharget that network automatically

Reply:
Can we open a ticket with these comments in it some we can have a later README update that provides documentation to help in this scenario